PhysicsTools/SelectorUtils/python/tools/vid_id_tools.py

```diff
@@ -125,7 +125,14 @@ def setupVIDMuonSelection(process,cutflow,patProducer=None):
     moduleName = "muoMuonIDs"
     if not hasattr(process, moduleName):
         raise Exception("VIDProducerNotAvailable", "%s producer not available in process!" % moduleName)
-    setupVIDSelection(getattr(process, moduleName), cutflow)
+    #touch cfg if the data format is MiniAOD
+    module = getattr(process, moduleName)
+    srcLabel = module.physicsObjectSrc.getModuleLabel()
+    if srcLabel == "slimmedMuons":
+        for p in cutflow.cutFlow:
+            if hasattr(p, 'vertexSrc'): p.vertexSrc = "offlineSlimmedPrimaryVertices"
```

Comment:
@jhgoh This will cause the md5 hash to change and make the code print errors. Please fix this.
I suggest you do something like what we do for EGM, where we configure for both.

Example python:
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/RecoEgamma/ElectronIdentification/python/Identification/heepElectronID_tools.py#L197-L207

Example C++:
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/RecoEgamma/ElectronIdentification/plugins/cuts/GsfEleDxyCut.cc#L49-L54